src/DockWidgetTab.h

```diff
@@ -136,6 +136,12 @@ private slots:
 	 */
 	bool isClosable() const;
 
+
+	/**
+	* Emits tooltipChanged
```

Comment:
In the implementation of the event function there is no tooltipChanged event emitted. Is this comment still valid?